integrationtest/ValidationProject/ValidateStrykerResults.cs

```diff
@@ -67,7 +67,7 @@ public void NetCoreWithTwoTestProjects()
 
             var report = JsonConvert.DeserializeObject<JsonReport>(strykerRunOutput);
 
-            CheckReportMutantCounts(report, total: 112, ignored: 6, survived: 10, killed: 14, timeout: 2, nocoverage: 78);
+            CheckReportMutantCounts(report, total: 112, ignored: 27, survived: 8, killed: 8, timeout: 2, nocoverage: 65);
```

Comment:
Can you verify these extra ignored mutants are all block type?

Reply:
I checked the reports, it looks like they are indeed block mutants.